CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -319,23 +322,25 @@ private ToolBar initToolbar() {
 
     private ContextMenu initColumnVisibilityContextMenu() {
         final ContextMenu cm = new ContextMenu();
-        final ArrayList<CustomMenuItem> columnCheckboxes = new ArrayList<>();
-        
-        final Label columnFilterLabel = new Label("Filter:");
-        final TextField columnFilterTextField = new TextField ();
-        final HBox filterBox = new HBox();
-        filterBox.getChildren().addAll(columnFilterLabel, columnFilterTextField);
-        
-        final CustomMenuItem columnFilter = new CustomMenuItem(filterBox);
-        columnFilter.setHideOnClick(false);
-        columnFilterTextField.setOnKeyReleased(event -> {
-            final String filterTerm = columnFilterTextField.getText().toLowerCase().trim();
-            columnCheckboxes.forEach(item -> {
-                final String columnName = item.getId().toLowerCase();
-                item.setVisible(filterTerm.isBlank() || columnName.contains(filterTerm));
-            });
-        });
-        
+        final ArrayList<CustomMenuItem> columnCheckboxesSource = new ArrayList<>();
+        final ArrayList<CustomMenuItem> columnCheckboxesDestination = new ArrayList<>();
+        final ArrayList<CustomMenuItem> columnCheckboxesTransaction = new ArrayList<>();
```

Comment:
For these 3, it is better to declare them simply as Lists and instantiate as an arraylist.
i.e. `final List<CustomMenuItem> foo = new ArrayList<>();`

Reply:
Fixed.